src/clj_kondo/impl/linters/keys.clj

```diff
@@ -5,13 +5,16 @@
    [clj-kondo.impl.utils :refer [node->line tag]]))
 
 (defn key-value
-  "We only support tokens as key values for now."
+  "We only support tokens, vectors, quoted forms and lists as key values for now."
   [node]
   (case (tag node)
     :token (or (when-let [v (:k node)]
                  (if (:namespaced? node)
                    (str v) v))
                (str node))
+    :vector (map key-value (:children node))
+    :list (map key-value (:children node))
+    :quote (recur (first (:children node)))
```

Comment:
Took the first of the children here — I'm not sure in what case a `quote` node can have multiple children.

Reply:
I think that is the correct solution, thank you.